config/configgrpc/configgrpc.go

```diff
@@ -176,6 +190,21 @@ func (gcs *GRPCClientSettings) ToDialOptions() ([]grpc.DialOption, error) {
 		opts = append(opts, keepAliveOption)
 	}
 
+	if gcs.PerRPCAuth != nil {
+		if strings.EqualFold(gcs.PerRPCAuth.AuthType, PerRPCAuthTypeBearer) {
+			sToken := gcs.PerRPCAuth.BearerToken
+			token := BearerToken(sToken)
+			if strings.HasPrefix(sToken, "file://") {
```

Comment:
Q.: does it really need to support a file? I mean the configuration already supports environment variables. If really needs to be a file the way to inject file content as configuration values should be standardized.

Reply:
In Kubernetes, it would be far easier to specify the location of the token file than to wrap the OpenTelemetry collector into a shell that would read the file and store its contents in an env var.

Reply:
Fair enough. My concern here is that we need to have a consistent way to handle config value injection via file. We are introducing it here but it will be good to have a standard way to do it. In the future, we may have to change it to support whatever becomes the standard way.

cc @bogdandrutu @tigrannajaryan @jrcamp

Reply:
I agree with @pjanotti
I think we will eventually have to add a generic mechanism for reading configuration portions from external files (or elsewhere). Adding this here is likely going to be different from that generic mechanism and will break the uniformnes of the config.
I'd avoid this for now and let if be handled by the generic config inclusion when it is implemented.

Reply:
Alright, I removed it for now, as I can (probably) use the downward API in Kubernetes as a workaround.